swarm/api/http/server.go

```diff
@@ -975,7 +976,38 @@ func (s *Server) HandleGetFile(w http.ResponseWriter, r *Request) {
 	}
 
 	w.Header().Set("Content-Type", contentType)
-	http.ServeContent(w, &r.Request, "", time.Now(), reader)
+	http.ServeContent(w, &r.Request, "", time.Now(), newBufferedReadSeeker(reader, getFileBufferSize))
+}
+
+// The size of buffer used for bufio.Reader on LazyChunkReader passed to
+// http.ServeContent in HandleGetFile.
+// Warning: This value influences the number of chunk requests and chunker join goroutines
+// per file request.
+// Recommended value is 4 times the io.Copy default buffer value which is 32kB.
+const getFileBufferSize = 4 * 32 * 1024
```

Comment:
this constant looks suspiciously low to me. I think your experiments were influenced by the overall number of peers, if there are more peers in a row (parctically in PO 0) then  i bet we can afford a higher readahead buffer. 
Lets leave it as is and make sure we experiment with this after chunk refactor

Reply:
Thanks Viktor. This surely needs more experimenting and testing.